packages/dev/core/src/AudioV2/abstractAudio/audioEngineV2.ts

```diff
@@ -0,0 +1,165 @@
+import type { Nullable } from "../../types";
+import type { AbstractAudioNode, AbstractNamedAudioNode } from "./abstractAudioNode";
+import type { MainAudioBus } from "./mainAudioBus";
+import type { AbstractSpatialAudioListener, ISpatialAudioListenerOptions } from "./subProperties/abstractSpatialAudioListener";
+
+const Instances: AudioEngineV2[] = [];
```

Comment:
A general issue that people will face (if not fully documented correctly) is that this array will grow more and more, unless the old audio engines are disposed before creating a new one.

Is there a reason to have more than one audioengine in the same context? if yes, maybe there should be a default parameter (set to true) to "reuse already-existing audio engine if available` when creating a new audio engine? 

I am thinking of a playground user who will create a new audio engine on each "run" call, not knowing that this array grows every time. eventually it will be a memory/perf issue. 

I know the audio engine is standalone, this is why i don't suggest connecting it to the (GL)Engine's onDispose.

Reply:
Would it make sense to add a `reuseExistingEngine` option to the existing `options` parameter instead of adding a new parameter?

Reply:
In the case of a playground, we will need to add internal playground code to dispose existing audio engines when the playground restarts, similar to how the graphics engine is disposed. This will make adding a `reuseExistingEngine` option unnecessary.